src/vm/tests/simple_apply_eval.rs

```diff
@@ -609,13 +613,17 @@ fn test_bad_lets() {
         "(let ((tx-sender 1)) (+ tx-sender tx-sender))",
         "(let ((* 1)) (+ * *))",
         "(let ((a 1) (a 2)) (+ a a))",
-        "(let ((a 1) (b 2)) (var-set cursor a) (var-set cursor (+ b (var-get cursor))) (+ a b))"];
+        "(let ((a 1) (b 2)) (var-set cursor a) (var-set cursor (+ b (var-get cursor))) (+ a b))",
+        "(let ((true 0)) true)",
+        "(let ((false 1)) false)"];
 
     let expectations: &[Error] = &[
         CheckErrors::NameAlreadyUsed("tx-sender".to_string()).into(),
         CheckErrors::NameAlreadyUsed("*".to_string()).into(),
         CheckErrors::NameAlreadyUsed("a".to_string()).into(),
-        CheckErrors::NoSuchDataVariable("cursor".to_string()).into()];
+        CheckErrors::NoSuchDataVariable("cursor".to_string()).into(),
+        CheckErrors::NameAlreadyUsed("true".to_string()).into(),
```

Comment:
Throwing `NameAlreadyUsed` is probably not optimal in terms of DX (but it's consistent with the errors currently in place). Should we create an issue for this @kantai?

Reply:
Yep, let's open an issue